src/app/(main)/my-voyage/[teamId]/ideation/ideationService.ts

```diff
@@ -53,23 +59,43 @@ export interface IdeationVoteResponse extends IdeationResponse {
   projectIdeaId: number;
 }
 
+const nextJsRestApiAdapter = new NextJsRestApiAdapter(
```

Comment:
Same here, do we prefer to keep all the types/interfaces in the same file with the implementation?

Reply:
I'm not really sure what you mean by this

Reply:
Like I mentioned before, I prefer keeping types/interfaces separate from the actual implementation/logic